test/test_datasources.py

```diff
@@ -150,7 +148,7 @@ def test_get_timeseries(self, gridadmin_result_mock):
         nds = NetcdfGroundwaterDataSource()
         m = mock.MagicMock()
         nds._ds = m
-        nds.get_timeseries("nodes", 3, "s1")
+        nds.get_timeseries("s1", 3)
```

Comment:
Wordt hier iets getest? Er worden drie dingen gemockt, maar ik zie zo gauw niet wat ermee gedaan wordt.

Reply:
Ik heb de initiële test niet geschreven en wat er precies getest wordt is me ook niet helemaal duidelijk. Hij crashed niet bij het aanroepen van de methode?